iot_packet_verifier/src/burner.rs

```diff
@@ -124,87 +126,89 @@ impl Burner {
                 return Ok(());
             };
 
-        // Fetch escrow account
-        let ddc_key = pdas::delegated_data_credits(&self.program_cache.sub_dao, &payer);
-        let account_data = self.provider.get_account_data(&ddc_key).await?;
-        let mut account_data = account_data.as_ref();
-        let escrow_account =
-            DelegatedDataCreditsV0::try_deserialize(&mut account_data)?.escrow_account;
+        if self.enable_dc_burn {
+            tracing::info!("Burning {} DC from {}", amount, payer);
```

Comment:
Can we the body of this if to another function so it is easier to tell the scope of the if?

Reply:
Yup